core/src/main/resources/error/error-classes.json

```diff
@@ -39,6 +46,10 @@
     "message" : [ "Found duplicate keys '%s'" ],
     "sqlState" : "23000"
   },
+  "FAILED_COMPILE_MSG" : {
+    "message" : [ "failed to compile: %s" ],
+    "sqlState" : "42000"
```

Comment:
I'm not sure if this is a syntax error, but I also can't find a relevant SQLSTATE. Maybe we can leave the SQLSTATE empty for now?